webapp/Connector/src/app/store/Publication.js

```diff
@@ -151,7 +151,7 @@ Ext.define('Connector.app.store.Publication', {
             }, this);
 
             // map the docs to each publication
-            let publicationMap = {};
+            var publicationMap = {};
```

Comment:
just curious, was there a problem with the production build that led to this change?

Reply:
Hi Karl - I was getting a compile-time error while building scss (changes in which i ended up not using). That was the only reason for the change.